src/server.py

```diff
@@ -20,14 +21,35 @@ def server():
                 msg += part
                 if len(part) < buffer_length:
                     break
```

Comment:
If using break, setting a pointer of 'message_complete = False' is unnecessary, your while loop can be set up using while True: